sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -3627,6 +3628,24 @@ class DataSourceV2SQLSuiteV1Filter
     }
   }
 
+  test("SPARK-49211: V2 Catalog can also support built-in data sources") {
+    def checkParquet(tableName: String): Unit = {
+      withTable(tableName) {
+        sql("CREATE TABLE " + tableName +
+          " (name STRING) USING PARQUET LOCATION '/tmp/test_path'")
```

Comment:
can we use `withTempPath` to generate the table path?

Reply:
Done